### ISSUE TITLE
Add FAQ Schema to Accordion Widget for better SEO results

### ISSUE THREAD

**Issue by christianbrucki** [elementor/elementor#13425]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The accordion widget is used on many websites for the FAQ page. But for best SEO results it is recommended to add a FAQ Schema. This allows to show questions and answers directly in the search results. As nearly every website has a FAQ page, I am wondering why such a powerful SEO tool is not integrated into the accordion widget?

**Describe the solution you'd like**
There are several third party addons which integrated that feature into an accordion. The best example I could found was the FAQ widget from Ultimate Addons. In the widget settings you can decide if you want to enable schema support or not. Another example would be the FAQ Widget from PowerPack or the Questionar FAQ Accordion from Merkulove. But as an Elementor Pro user I think there should be no need to buy an extra addon, as it is such an essential function.

**Additional context**
For further details about the FAQ Schema can be found at Google Search Central: https://developers.google.com/search/docs/data-types/faqpage?hl=en

**Comment by DorShahar** [comment]
Hey all,

This issue has been resolved in Elementor v3.2.1

Feel free to update!

**Comment by djianbooth** [comment]
Hello. please consider including a node identifier (JSON @id) to this FAQpage schema object so as to allow linked data.

https://sitebulb.com/resources/guides/node-identifiers-from-structured-data-to-linked-data/

**Comment by trinermedia** [comment]
@djianbooth is right. Otherwise you will get an error message from Google Search Console if you place several accordion widgets with FAQ schema on one page, as you can see here: https://rankmath.com/kb/fix-duplicate-field-faqpage-error/
